example-gto/code/.github/workflows/gto-act-on-tags.yml

```diff
@@ -0,0 +1,46 @@
+name: Act on artifact registrations and promotions
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  act:
+    name: Check
```

Comment:
what does it check (I can't understand the logic of this workflow tbh) - seems we have some things that are not needed, but I'm not sure

Reply:
That was a wrong name, renamed it. Removed extra stuff that's not needed.